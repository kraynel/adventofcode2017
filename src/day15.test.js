const { getFinalCount, getFinalCount2 } = require("./day15");

describe("getFinalCount", () => {
  it("should count", () => {
    expect(getFinalCount(65, 8921)).toBe(588);
  });

  it("should count", () => {
    expect(getFinalCount(703, 516)).toMatchSnapshot();
  });
});

describe("getFinalCount2", () => {
  it("should count 2", () => {
    expect(getFinalCount2(65, 8921)).toBe(309);
  });

  it("should count 2", () => {
    expect(getFinalCount2(703, 516)).toMatchSnapshot();
  });
});
