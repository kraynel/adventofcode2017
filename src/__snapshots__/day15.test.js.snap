// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getFinalCount should count 1`] = `594`;

exports[`getFinalCount2 should count 2 1`] = `328`;
